db/migrate/20210307213408_add_video_conference_schedule_to_open_studios_participant.rb

```diff
@@ -0,0 +1,5 @@
+class AddVideoConferenceScheduleToOpenStudiosParticipant < ActiveRecord::Migration[6.1]
+  def change
+    add_column :open_studios_participants, :video_conference_schedule, :text
```

Comment:
mysql doesn't have `json` or `array` columns so we just use text and let Rails take care of the rest with `store`